webapp/src/main/webapp/news-details/components/ExoNewsDetails.vue

```diff
@@ -264,7 +263,20 @@ export default {
           window.location.href = this.news.spaceUrl;
         }
       }, redirectionTime);
-    }
+    },
+    targetBlank: function(content) {
+      let internal = location.host.replace('www.', '');
```

Comment:
I think that it not needed to replace www and ignore case. the link host should begin exactly with location.host + eXo.env.portal.context